src/middleware/cache.js

```diff
@@ -0,0 +1,121 @@
+'use strict';
+
+// Note - private/public declarations will automatically be added as appropriate if not specified
+const presets = {
+	no: {
+		maxAge: 0,
+		conditions: 'no-cache, no-store, must-revalidate'
```

Comment:
Ah I misunderstood you there. This is also used to set the outbound cache header. Perhaps should be stored in a different variable though